dashboard/config/routes.rb

```diff
@@ -216,6 +216,10 @@
   put '/featured_projects/:project_id/unfeature', to: 'featured_projects#unfeature'
   put '/featured_projects/:project_id/feature', to: 'featured_projects#feature'
 
+  get '/weblab/footer', to: 'projects#weblab_footer'
```

Comment:
Could we add a comment here to indicate that these routes are used for weblab shares on codeprojects.org?